actionlist/index.ios.js

```diff
@@ -1,9 +1,10 @@
+/* eslint-disable */
```

Comment:
Why are you disabling linting on an entire file? ESLint supports both single-line and rule-specific disables. In either case, any time you disable linting there should be a very good explanation in the comments to describe why.